### DISCUSSION TITLE
Allow more than one filter panel per search results

### DISCUSSION THREAD

**Discussion by PapaGeil** [microsoft-search/pnp-modern-search#4059] [Ideas]
Hi,

I'd like to suggest making possible to add more than one filters web part to a search results web part. There may be a few benefits to this:

When implementing two different sets of filters that are available through verticals, one now has to duplicate the search results web part. It would be much easier if the same search results would be connected to both filter web parts. Example shows simple and advanced filter settings with "simple" only including the most relevant filters

![image](https://github.com/user-attachments/assets/ed66a13c-45d6-462f-a5dc-c425c36be767)

Also, when there are many filters available to the user, allowing multiple filter web parts would make it easier to navigate through the filters available by adding headings:

![image](https://github.com/user-attachments/assets/a91808f5-c18b-439d-9059-4f11184d29a7)

Thank you for considering this sugestion!

**Comment by kasperbolarsen** [comment]
We are absolutely happy to review a PR from you to that effect 😀
